### PR TITLE
UI/card tiles front end

### LINKED ISSUE #87: UI/Card-Tiles-Front-End
- [ ] Update home screen to card tiles  (current spaces, add space, create space to card tiles)
image
- [ ] Update current space/all ideas to card-tiles
image

NEW:
image

### PR DESCRIPTION
**Pull request by kimberlym4488**
Test:
Try logging in as janedoe and johnsmith to see status differences on different spaces and ideas.  

Notes:
The like functionality is disabled on the space view. 
A user can view the # of likes. It worked fine on desktop but on mobile there is no 'hover' effect. 

So the user on mobile only sees the static version of the card.  The entire card is clickable to take the user through to the idea.

Mobile View:
image

Walk-through
https://www.loom.com/share/88de2d48044e42baaaa6e70c98068f93

Resolves #87

### CODE REVIEW

File: views/partials/spaceMini.handlebars [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
+                class="overlay-hover inner-down-hover bg-primary text-white shadow " style="--bs-bg-opacity: 0.85">
+
+                    <span class="d-block px-4 py-3 fs-4 fw-bold text-center">
+                        View Space<br/>
+                        <span class="fw-normal fs-5">{{space.name}}</span>
+                    </span>
+                </a>
+                <p class="m-0 overlay-hover--hide inner-down-hover--also fs-4 d-block text-center">
+                  {{#if (is_match status 'approved')}}
+                    <span class="badge fs-5 bg-primary bi-person-workspace">  {{space.name}}</span>
```
Comment (ac524) [inline views/partials/spaceMini.handlebars:22]: Space name badges need the class `text-wrap` to make sure long names wont shoot out the box

File: views/partials/ideaMini.handlebars [hunk 2]
```diff
@@ -1,20 +1,36 @@
...
+          Check out this<br /></span>
+        <span class='fw-normal fs-5'>Idea</span>
+      </span>
+    </a>
+
+    {{! </a> }}
+    <p
+      class='m-0 overlay-hover--hide inner-down-hover--also fs-4 d-block text-center text-wrap'
+    >
+      <span class='badge fs-5 bg-primary'>
```
Comment (ac524) [inline views/partials/ideaMini.handlebars:19]: Same here. Name badges need the class `text-wrap` to make sure long names wont shoot out the box

**Review by ac524 — CHANGES_REQUESTED** [review]
Just need a class on some `badge` elements :)

File: views/partials/spaceMini.handlebars (hunk 1 above)
Comment (kimberlym4488) [reply views/partials/spaceMini.handlebars]: Thanks!

File: views/partials/ideaMini.handlebars (hunk 2 above)
Comment (kimberlym4488) [reply views/partials/ideaMini.handlebars]: All set!

**Review by ac524 — APPROVED** [review]
🙌

**Pull request merged**